gui/slick/js/rootDirs.js

```diff
@@ -181,8 +181,8 @@ $(document).ready(function() {
     });
 
     $('#defaultRootDir').on('click', function(){
-        if ($("#rootDirs option:selected").length) {
-            setDefault($("#rootDirs option:selected").attr('id'));
+        if ($('#rootDirs option:selected').length) {
+            setDefault($('#rootDirs option:selected').attr('id'));
```

Comment:
It should be reading from prop as attr isn't 100% reliable as it relies on the DOM and is slow..